File: src/main/java/zwylair/randomitembattle/commands/TimeoutCommand.java
package zwylair.randomitembattle.commands;

import com.mojang.brigadier.CommandDispatcher;
import com.mojang.brigadier.arguments.IntegerArgumentType;
import com.mojang.brigadier.context.CommandContext;

import net.minecraft.text.Text;
import net.minecraft.server.command.ServerCommandSource;
import static net.minecraft.server.command.CommandManager.argument;
import static net.minecraft.server.command.CommandManager.literal;

import static zwylair.randomitembattle.RandomItemBattle.*;


public class TimeoutCommand {

    public static void register(CommandDispatcher<ServerCommandSource> dispatcher) {
        dispatcher.register(literal("rib").then(literal("timeout").then(argument("seconds", IntegerArgumentType.integer())
                .executes(TimeoutCommand::execute))));
    }

    public static int execute(CommandContext<ServerCommandSource> ctx) {
        final int timeoutInSeconds = IntegerArgumentType.getInteger(ctx, "seconds");
        TIMEOUT_IN_TICKS = timeoutInSeconds * 20;

        ctx.getSource().sendFeedback(() -> Text.literal(String.format(chatModPrefix + "Timeout set to %s", timeoutInSeconds)), false);
        return 0;
    }
}

File: src/main/kotlin/zwylair/randomitembattle/commands/TimeoutCommand.kt
package zwylair.randomitembattle.commands

import com.mojang.brigadier.CommandDispatcher
import com.mojang.brigadier.context.CommandContext
import com.mojang.brigadier.arguments.IntegerArgumentType

import net.minecraft.text.Text
import net.minecraft.server.command.ServerCommandSource
import net.minecraft.server.command.CommandManager.argument
import net.minecraft.server.command.CommandManager.literal

import zwylair.randomitembattle.RandomItemBattle

object TimeoutCommand {
    fun register(dispatcher: CommandDispatcher<ServerCommandSource>) {
        dispatcher.register(literal("rib").then(literal("timeout").then(
            argument("seconds", IntegerArgumentType.integer())
                .executes { execute(it) } )
        )) }

    private fun execute(ctx: CommandContext<ServerCommandSource>): Int {
        val timeoutInSeconds = IntegerArgumentType.getInteger(ctx, "seconds")
        RandomItemBattle.TIMEOUT_IN_TICKS = timeoutInSeconds * 20

        ctx.source.sendFeedback({ Text.of("${RandomItemBattle.CHAT_PREFIX_MOD} Timeout set to $timeoutInSeconds") }, false)
        return 0
    }
}
